config/webpack.demo.config.js

```diff
@@ -38,15 +38,19 @@ module.exports = {
   },
   resolve: {
     extensions: ['.tsx', '.ts', '.js'],
+    fallback: {
+      path: false,
+      crypto: false,
```

Comment:
@evanpurkhiser Pretty sure we shouldn't fallback on these libraries cause I can't imagine why we'd need them on the browser lol

Reply:
you'd be surprised, but this seems fin yeah